### PR TITLE
[MVI-798] (requests-mv-integrations) Expose response url

### PR DESCRIPTION
**Pull request by jefftune**
Expose at DEBUG level, built URL from GET, POST, and PUT responses.

### CODE REVIEW

**Comment by tulitune** [comment]
@jefftune - Functionality wise looks great.
In terms of style ( I've been looking at this code in the last days ), we have a long if-else statement,
which paragraphs are very similar. I guess we can make a function, and call it with different parameters. 
Anyway, it is not import functionality wise, so I think we are good to fo with this merge.
We can brainstorm if we wish to refactor the code and how, and open another ticket.

File: requests_mv_integrations/request_mv_integration.py [hunk 1]
```diff
@@ -885,6 +885,14 @@ def _request(
 
                 response = self.tune_request.request(**kwargs)
 
+                if hasattr(response, 'url'):
```
Comment (zaquestion) [inline requests_mv_integrations/request_mv_integration.py]: The necessity for a `hasattr` here concerns me. Is there another option for the response?

I don't believe so peaking at the code.

File: requests_mv_integrations/request_mv_integration.py [hunk 2]
```diff
@@ -885,6 +885,14 @@ def _request(
 
                 response = self.tune_request.request(**kwargs)
```
Comment (zaquestion) [inline requests_mv_integrations/request_mv_integration.py]: I realize not part of the changes here, but I think the usage of tune_request.request here allows os to combine the (below) into one block?
```
 if request_method == 'GET':
elif request_method == 'POST':
...
```

**Comment by jefftune** [comment]
@zaquestion 
I have seen that if `requests` fails, it's `response` does not necessarily always has `url`

**Pull request merged**